architecture/infrastructure_components/image_registry.adoc

```diff
@@ -46,11 +45,24 @@ creation.  Refreshing the fetched tags is as simple as running `oc import-image
 <stream>`. When new images are detected, the previously-described build and
 deployment reactions occur.
 
-[[authentication]]
+[[red-hat-quay]]
+== Red Hat Quay Registries
+
+If you need an enterprise-quality container image registry, Red Hat Quay is available both as a hosted service and as software you can install in your own data center or cloud environment. Advanced registry features in Red Hat Quay include geo-replication, image scanning, and the ability to rollback images.
+
+Visit the link:https://quay.io[Quay.io] site to set up your own hosted Quay registry account. After that, the link:https://quay.io/tutorial/[Quay Tutorial] helps you login to the Quay registry and start managing your images. Alternatively, refer to link:https://access.redhat.com/documentation/en-us/red_hat_quay/2.9/html-single/getting_started_with_red_hat_quay/[Getting Started with Red Hat Quay] for information on setting up your own Red Hat Quay registry.
 
+At the moment, you access your Red Hat Quay registry from OpenShift as you would any remote container image registry. To learn how to set up credentials to access Red Hat Quay as a secured registry, refer to xref:../../dev_guide/managing_images.adoc#allowing-pods-to-reference-images-from-other-secured-registries[Allowing Pods to Reference Images from Other Secured Registries].
```

Comment:
s/At the moment, you access your Red Hat Quay registry from OpenShift as you would any/You can access your Red Hat Quay registry from {product-title} like any